README.md

```diff
@@ -1 +1,174 @@
-# ROR-Capstone-Budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h2><b>Budget App</b></h2>
+  <h4>Jose Berbesi</h4>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget Boss <a name="about-project"></a>
+
+**Budget App** is a Ruby on Rails mobile web application where users can manage their budget: they have a list of transactions associated with a category, so that users can see how much money they spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Creating data models**
+- **Creating controllers**
+- **Validations and Model specs.**
+- **Add Devise.**
+- **Add unit rspec and integration Capybara tests.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ The link to the project deployed and the presentation video:
+
+- [Live Demo Link](https://mybudgetappbyjose.onrender.com/)
+- [Project presentation video]( https://www.loom.com/share/78b89e62fc4349bdb98de4fd4518c1e1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Great job on your video presentation, kindly ensure you clearly highlight a piece of code you found interesting, because you ran out of time this was not clearly stated. Kindly redo your video